src/base/bittorrent/sessionimpl.h

```diff
@@ -430,6 +433,14 @@ namespace BitTorrent
         void findIncompleteFiles(const TorrentInfo &torrentInfo, const Path &savePath
                                  , const Path &downloadPath, const PathList &filePaths = {}) const;
 
+        void enablePortMapping();
+        void disablePortMapping();
+        void addMappedPorts(const QSet<quint16> &ports);
+        void removeMappedPorts(const QSet<quint16> &ports);
+
+        void invoke(std::function<void ()> func);
```

Comment:
This method seems to be unused.

Reply:
>This method seems to be unused.

It was used initially, but then I slightly changed the solution of some subtask. But it is still used in the following PR.